server/metrics/metrics.go

```diff
@@ -906,6 +917,42 @@ var (
 		Help:      "Per-file upload duration during remote execution, in **microseconds**.",
 	})
 
+	// FirecrackerStageDurationMsec For a firecracker container assigned to
+	// execute a task, tracks the duration of each stage
+	//
```

Comment:
(sorry, being extra nit-picky here since this will show up in the website docs)

this part can be removed since the website docs will show the 'help' text of the metric.

```suggestion
```